Duet/ProjectR3D/300ZL/config.g

```diff
@@ -18,10 +18,10 @@ M83                                 	; ...but relative extruder moves
 M667 S1								; CoreXY mode
 
 M584 X0 Y1 Z5:6:7 E3:4:8:9 		; Map Z to drivers 5, 6, 7. Define unused drivers 3,4,8 and 9 as extruders
-M569 P0 S1                          ; Drive 0 goes forwards (change to S0 to reverse it) X stepper
-M569 P1 S0                          ; Drive 1 goes backwards	Y Stepper
+M569 P0 S0                          ; Drive 0 goes forwards (change to S0 to reverse it) X stepper
```

Comment:
There are many reasons why the steppers might go in a different direction (wiring, etc), so I think it is best to standardize on the Project R3d config.

See: https://github.com/railcore/configs/blob/master/Duet/ProjectR3D/300ZL/config.g#L21-L28